### PR TITLE
Complete translation to simplified Chinese.

### PR DESCRIPTION
**Pull request by Gosin**
I have translated all files except `tz.php`.

### CODE REVIEW

**Comment by fetus-hina** [comment]
I haven't been able to decide if the translation should reflect the names of proper nouns such as weapons or stages.
I know that at least some Chinese (including traditional Chinese) speakers have used the English version of game copy.
I also know that some of them use the Japanese version.

May be their "maximum common denominator of happiness" is to keep their English names.
(There's a statement on the wiki based on this premise, I wrote.)

If your translation is already widely used in the Chinese community, I have no objection to it.

File: messages/zh-CN/app.php [hunk 1]
```diff
@@ -4,622 +4,624 @@
...
+    'User Name' => '用户名字',
+    'Users' => '用户',
+    'Use your profile icon' => '使用档案头像',
+    'Verification Code' => '验证码',
+    'Verify' => '验证',
+    'Version' => '版本',
+    'Version {0}' => '版本 {0}',
+    'View Settings' => '显示设置',
+    'We\'ll send an email when you log in to the website or change your password.' => '登录网站或是更改密码后我们会给你发送一封邮件。',
     'We\'ll send an email when you log in to the website or change your password.' => '',
```
Comment (fetus-hina) [inline messages/zh-CN/app.php:548]: Could you remove this line?

File: messages/zh-CN/app.php [hunk 2]
```diff
@@ -4,622 +4,624 @@
...
+    '{0} etc.' => '{0}等',
+    '{0}×{1} or less resolution' => '{0}×{1} 或更低的分辨率',
+    '{attribute} must be at most 15 alphanumeric or underscore characters.' => '{attribute} 最多是15个字母或数字或下划线。',
+    '{decimal5_7} Format' => '{decimal5_7} 格式',
+    '{lower}-{upper} percentile' => '{lower}-{upper} 百分位数',
+    '{n,plural,=1{battle} other{battles}}' => '{n,plural,=1{对战} other{对战}}',
+    '{nFormatted} {n, plural, =1{time} other{times}}' => '{nFormatted} {n, plural, =1{次} other{次}}',
+    '{number, plural, =1{1 death} other{# deaths}} in {battle, plural, =1{1 battle} other{# battles}}' => '{number, plural, =1{一次死亡} other{#次死亡}} in {battle, plural, =1{一场对战} other{#场对战}}',
+    '{number, plural, =1{1 egg} other{# eggs}} in {battle, plural, =1{1 shift} other{# shifts}}' => '{number, plural, =1{一个蛋} other{#个蛋}} in {battle, plural, =1{1次打工} other{#次打工}}',
     '{number, plural, =1{1 kill} other{# kills}}' => '',
```
Comment (fetus-hina) [inline messages/zh-CN/app.php:608]: Could you update to `'{number, plural, =1{1 kill} other{# kills}}' => '{number} 杀敌',` or something?

File: messages/zh-CN/app.php [hunk 3]
```diff
@@ -4,622 +4,624 @@
...
-    '~{point}p' => '',
+    '{number, plural, =1{1 kill} other{# kills}} in {battle, plural, =1{1 battle} other{# battles}}' => '{number, plural, =1{一次杀敌} other{#次杀敌}} in {battle, plural, =1{一次对战} other{#次对战}}',
+    '{number, plural, =1{1 time} other{# times}} in {battle, plural, =1{1 shift} other{# shifts}}' => '{number, plural, =1{1次} other{#次}} in {battle, plural, =1{一次打工} other{#次打工}}',
+    '{pct_point} percentage point' => '{pct_point}百分点数',
+    '{pct} % loss' => '{pct}%失败率',
+    '{pct} ({dupf} DU/F)' => '{pct} ({dupf} 单位距离/帧)',
+    '{percentile} Percentile' => '{percentile}百分位数',
+    '{point, plural, =1{1 point} other{# points}} in {battle, plural, =1{1 battle} other{# battles}}' => '{point, plural, =1{一点} other{#点}} in {battle, plural, =1{一场对战} other{#场对战}}',
+    '{point, plural, other{#p}}' => '{point, plural, other{#点}}',
+    '{point}p' => '点',
```
Comment (fetus-hina) [inline messages/zh-CN/app.php:617]: Please update to `'{point}p' => '{point}点',`

File: messages/zh-CN/app.php [hunk 4]
```diff
@@ -4,622 +4,624 @@
...
+    '{rule}: Current' => '{rule}:现在',
+    '{rule}: Peak' => '{rule}: 最高',
+    '{sec,plural,=1{# second} other{# seconds}}' => '{sec,plural,=1{#秒} other{#秒}}',
+    '{sec} seconds' => '秒',
```
Comment (fetus-hina) [inline messages/zh-CN/app.php:621]: Please update to `'{sec} seconds' => '{sec}秒',`

File: messages/zh-CN/app.php [hunk 5]
```diff
@@ -4,622 +4,624 @@
...
+    '{sec} seconds ({pct} %)' => '{sec} 秒 ({pct} %)',
+    '{sec} seconds, {cnt} times' => '{sec} 秒, {cnt} 次',
+    '{theme} Theme' => '{theme}主题',
+    '{title} Official Website' => '{title}官方站点',
+    '~{point}p' => '~{point}点',
```
Comment (fetus-hina) [inline messages/zh-CN/app.php:626]: The "~" means "about".
"~1000p" is translated into Japanese as "約1000p".
I think in the Chinese language is probably same (大约1000点 or something?), what do you think?

**Comment by Gosin** [comment]
> Perhaps "median" is "中位置" (in Japanese, "中央値")

Yes, it should be “中位值” instead. I had a typo here.

**Comment by Gosin** [comment]
> Perhaps the translation of "mode" is not correct.
> "Mode" is the "most frequently appearing value" (in Japanese, "最頻値").
> 
> For example, the dataset is "1 1 2 3 4 5 6",
> min = 1
> max = 6
> average = 3.143
> median = 3
> mode = 1 (because only "1" appears twice)

Right, I didn't get it right. It should be “众数” instead。

**Comment by Gosin** [comment]
> The "MMMM y" is a format for the year and month (e.g., "January 2020") in [ICU format](http://userguide.icu-project.org/formatparse/datetime#TOC-Date-Time-Format-Syntax).
> This format is probably "2020年1月" in Chinese, the same as in Japanese.
> If so, this sould be `'y\'年\'M\'月\''`.

Yes, I'll update it.

**Comment by Gosin** [comment]
> I haven't been able to decide if the translation should reflect the names of proper nouns such as weapons or stages.
> I know that at least some Chinese (including traditional Chinese) speakers have used the English version of game copy.
> I also know that some of them use the Japanese version.
> 
> May be their "maximum common denominator of happiness" is to keep their English names.
> (There's a statement on the wiki based on this premise, I wrote.)
> 
> If your translation is already widely used in the Chinese community, I have no objection to it.

Yes, it's really hard for me to decide, too. But according to posts and game videos from Chinese community. I think most of them are using the translations(meaning) of Japanese version. Therefore, I translated them based on Japanese version which I think should be accepted by most Chinese players.

**Comment by Gosin** [comment]
> If you don't need to express the plural (like Japanese lang.), you can use like `'{n,plural,=1{battle} other{battles}}' => '对战',`
> 
> https://github.com/fetus-hina/stat.ink/blob/3a8aa9a11d0984d27ccf15cbf5ef090814b30188/messages/ja/app.php#L604-L605

Cool, this is good to know.

File: messages/zh-CN/app.php (hunk 1 above)
Comment (Gosin) [reply messages/zh-CN/app.php:548]: Yes, I must accidentally added it when resolving merge conflicts.

File: messages/zh-CN/app.php (hunk 2 above)
Comment (Gosin) [reply messages/zh-CN/app.php:608]: Yes. Translated it to `'{battle}次对战中杀敌{number}次'`

File: messages/zh-CN/app.php (hunk 3 above)
Comment (Gosin) [reply messages/zh-CN/app.php:617]: Yes, updated.

File: messages/zh-CN/app.php (hunk 4 above)
Comment (Gosin) [reply messages/zh-CN/app.php:621]: Yes, updated

File: messages/zh-CN/app.php (hunk 5 above)
Comment (Gosin) [reply messages/zh-CN/app.php:626]: Yes, I agree. Translated to `约{point}点`

**Comment by fetus-hina** [comment]
Thank you for your hard work!
This pull-request has merged and will deploy soon.

**Pull request merged**